.github/workflows/e2e.yml

```diff
@@ -14,12 +14,12 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: "3.7"
-          cache: "pip"
```

Comment:
why removing the cache?

Reply:
It was necessary to make the test rework. I will try to reuse it